### PR TITLE
Better devito management

### PR DESCRIPTION
**Pull request by mloubout**
GPU support for the wave equation solves via devito's offloading.

**Compiler installation is up to the user and people should refer to the devito page/slack/github/ for GPU compiler issues**

### CODE REVIEW

**Comment by mloubout** [comment]
@ziyiyin97 pleas have a look and check this branch, is needed for newest devito

**Review by ziyiyin97 — COMMENTED** [review]
Looks nice! I am thinking whether there could be a terminal output which does something like

println("You are using $(configuration['language']) in $(configuration['platform'])") etc.

**Comment by mloubout** [comment]
That would be extremely verbose  since it would print it for every source and every operator

File: README.md [hunk 1]
```diff
@@ -17,6 +17,30 @@ You can find an FAQ with answers to issues at [FAQ](https://github.com/slimgroup
...
 
+## GPU
+
+JUDI supports the computation of the wave equation on GPU via [Devito](https://www.devitoproject.org)'s GPU offloading support.
+
+**NOTE**: Only the wave equation part will be computed on GPU, the julia arrays will still be CPU arrays and `CUDA.jl` is not supported.
+
+### Installation
+
+To enable gpu support in JUDI, you will need to install one of [Devito](https://www.devitoproject.org)'s supported offloading compilers. We strongly recommend checking the [Wiki](https://github.com/devitocodes/devito/wiki) for installation steps and to reach out to the Devito community for GPU compiler related issues.
```
Comment (ziyiyin97) [inline README.md:28]: This somehow contradicts the fact that Devito could be installed automatically if JUDI can't find it.

Comment (mloubout) [reply README.md:28]: There is a massive difference bettween installing Devito, that is the python package, and the the compiler (gcc, icc, ....) that ae used to compile the generated C code. And while JUDI will install Devito, it is up to the user to install proper compiler that are system and hardware dependent is up to the user.

**Review by ziyiyin97 — APPROVED** [review]

**Pull request merged**